### PR TITLE
Newsletters: add 53 (2019-07-03)

### PR DESCRIPTION
**Pull request by harding**
Todo:

- [x] Add URL for C-Lightning release (they've tagged final but haven't announced it yet, as of Sunday UTC)

### CODE REVIEW

**Comment by moneyball** [comment]
utACK - read through the entire newsletter and looks great!

File: _posts/en/newsletters/2019-07-03-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  pre-image to decrypt the data.  The system still requires the buyer
+  trust the merchant, as the merchant could deliver encrypted junk
+  instead of the actual data (i.e., this proposal isn't trustless like a
+  [zero-knowledge contingent payment][zkcp]), but the proposed protocol
+  can allow the buyer to begin downloading data while the payment is
+  still being processed.
+
+- **Lightning Loop supports user loop-ins:** as described in [Newsletter
+  #39][], [Lightning Loop][] uses *submarine swaps* to allow a user to
+  exchange bitcoins in an offchain LN payment channel for a bitcoins in
```
Comment (bitschmidty) [inline _posts/en/newsletters/2019-07-03-newsletter.md]: > for a bitcoins

to

> for bitcoins

File: _includes/specials/bech32/16-checklist.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+Recently, an Optech contributor surveyed many popular wallets and Bitcoin
```
Comment (jnewbery) [inline _includes/specials/bech32/16-checklist.md]: I think you need to update 2019-03-19-bech32-sending-support.md to include this.

File: _includes/specials/bech32/16-checklist.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+     $ p2wsh=$(
+       bitcoin-cli addmultisigaddress 1 \[$(
+         bitcoin-cli getaddressinfo $p2wpkh | jq .pubkey
+       )\] "bech32 test" bech32 | jq -r .address
+     )
+     $ echo $p2wpkh $p2wsh
+     $ echo $p2wpkh $p2wsh | tr '[a-z]' '[A-Z]'
+     ```
+
+2. Test sending a small amount of bitcoin to each lowercase address
```
Comment (jnewbery) [inline _includes/specials/bech32/16-checklist.md]: Should we add some additional warnings here:

- backup the wallet used to generate the addresses to make sure the funds don't get lost
- do this on testnet before trying on mainnet
- A final warning like "WARNING! If your software does not correctly implement sending to bech32 addresses these funds may become unspendable. Only send a minimal amount that you can afford to lose."

(may be overkill, but always better to be safe I think)

File: _includes/specials/bech32/16-checklist.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+4. Ensure that you received the funds by checking either the wallet you
+   used to create the addresses or a block explorer.  If that worked,
+   your software fully supports current bech32 spending addresses.
+
+     If you created addresses using a temporary Bitcoin Core wallet, you
+     can wait for the transactions to confirm and then send all the money
+     to your regular wallet using the following command: `bitcoin-cli
+     sendtoaddress YOUR_ADDRESS $( bitcoin-cli getbalance ) '' '' true`
+
```
Comment (jnewbery) [inline _includes/specials/bech32/16-checklist.md]: Do we also want wallets/services to test sending to v1+ segwit addresses? Probably not on mainnet, I'd imagine.

Comment (harding) [reply _includes/specials/bech32/16-checklist.md]: I would recommend that but it's not supported by a released version of Bitcoin Core yet (only master), so I don't think we should encourage it at this time.

File: _includes/specials/bech32/16-checklist.md (hunk 3 above)
Comment (harding) [reply _includes/specials/bech32/16-checklist.md]: I'm adding a line about "only using a small amount you can afford to lose."  I think the rest of these suggestions are unnecessary if that rule is followed.

**Comment by harding** [comment]
Pushed commits that should address all feedback that I didn't comment on.  Thanks everyone for the reviews!

**Comment by jnewbery** [comment]
ACK 2aab9660f11c555b6cc36b32ffa761d206150e83 (once C-Lightning release URL is added).

**Comment by harding** [comment]
Although C-Lightning did tag 0.7.1 several days ago, they haven't put it on their GitHub release page nor sent their usual mailing list announcement, so I've commented out that paragraph for tomorrow's newsletter.  @bitschmidty feel free to delete it or uncomment it (and add the required link) at merge time as you think is appropriate.  Here's their releases page: https://github.com/ElementsProject/lightning/releases ; if you click on the title of a release, it gives you a direct link---that's what I use for their releases URL.

In the meantime, LND just made their 0.7.1 release today, so I deleted the action item for their RC testing and gave them a news item for their release.

**Comment by harding** [comment]
C-Lightning just posted the release to their GitHub, so I added that back to this week's newsletter.

**Comment by bitschmidty** [comment]
tACK f1d2e40 
resolved PR discussions and squashed commits

**Comment by bitschmidty** [comment]
We are merged! Thanks @harding @moneyball @jnewbery

**Pull request merged**